translation/Dockerfile

```diff
@@ -11,11 +11,18 @@ RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     update-locale LANG=en_US.UTF-8
 ENV LANG en_US.UTF-8
 
-ADD opennmt-py/requirements.txt /opennmt_requirements.txt
-RUN apt-get install -y git default-jre && pip3.6 install -r opennmt_requirements.txt && pip3.6 install numpy -I
+RUN mkdir -p /singnet/translation/ && \
```

Comment:
I suggested a while ago it was a bad idea to `git pull` inside the Dockerfile instead of using the docker context.

One reason this is a bad idea is that this step will be cached. Docker has no way to know that this relies on an external resource (a git repo) and that this will have changed.

The docker context and my steps for adding the requirements as separate steps was to ensure it worked well with the docker cache. It will cache installing dependencies (since the `ADD` step is cached based on the contents of the added files).

Reply:
I don't think this is a bad idea, we have many approaches to handle docker cache. I don't like the idea to have a docker image depending of CircleCI to be built. The dockerfile should be able to be built without executing any step before start building. 

I've made some changes to ensure that we will cache all the requirements and get the last version of the services.

Now we can build a docker image from anywhere with no need third-party tools.